tendrl/commons/jobs/__init__.py

```diff
@@ -51,9 +51,7 @@ def _run(self):
 
                         # Job routing
                         if raw_job.get("payload", {}).get("tags", []):
-                            NS.node_context = NS.node_context.load()
-                            tags = json.loads(NS.node_context.tags)
-                            if set(tags).isdisjoint(
```

Comment:
You'll need to load() the node context because service tags are added during node sync in addition to tags mentioned in config. This change is not required